index.md

```diff
@@ -1,5 +1,39 @@
 ---
-title: About JuMP
+layout: page
 ---
 
-hello, world
+![jump](/assets/jump-logo-with-text-large.svg)
+
+JuMP is a modeling language and supporting packages for mathematical
+optimization in [Julia](https://julialang.org).
+
+## Getting started
+
+To get started, read the [Installation Guide](https://www.juliaopt.org/JuMP.jl/stable/installation/)
+in the JuMP documentation.
+
+Once you've installed JuMP, a good next step is the [Quickstart Guide](https://www.juliaopt.org/JuMP.jl/stable/quickstart/).
+
+There are also a number of tutorials available at [JuMPTutorials.jl](https://github.com/JuliaOpt/JuMPTutorials.jl).
+
+If you need help, search the [Forum](https://discourse.julialang.org/c/domain/opt)
+for similar questions, or open a new one.
+
+---
+
+##### 4th annual JuMP-dev
+
+We are pleased to announce that the 4th annual JuMP-dev workshop will be held
+June 15-17 2020 in [Louvain-la-Neuve, Belgium](https://goo.gl/maps/fyxqevrKn76nvhwq9),
+in conjunction with UCLouvain.
+
+Further details, including a call for speakers, will be announced shortly.
+
+For information on the previous JuMP-dev workshops, see:
+[2019](meetings/santiago2019/), [2018](meetings/bordeaux2018/), [2017](meetings/mit2017/).
+
+---
+
+- **Overview**: [presentation](https://www.youtube.com/watch?v=7LNeR299q88) and [workshop](https://www.youtube.com/watch?v=nnL7yLMVu6c) from JuliaCon 2015
```

Comment:
The JuliaCon presentation and the IJOC paper are old material. If we list it we should make it clear that it's outdated. Not content for the main page.